blog/content/post/2023-09-27-aem_phase2.md

```diff
@@ -126,17 +126,226 @@ goes without saying that the test should be conducted on the machine with TPM
+![Installing GRUB to disk and AEM package](/img/qubes_aem_v0.2_8.png)
+
+All packages are in place. Before we can proceed with provisioning AEM, we
+have to clear the TPM in the BIOS. Otherwise the next step may return the
+following error:
+
+![AEM TPM setup error](/img/qubes_aem_v0.2_9.png)
+
+Some platforms may require disabling Intel Trusted Execution Technology (TXT)
+in order to clear TPM. So after you clear the TPM, remember to enable Intel
+TXT back, otherwise AEM will not work. Once TPM is cleared, perform the TPM
+setup:
+
+```bash
+sudo anti-evil-maid-tpm-setup
+```
+
+![AEM TPM setup 1](/img/qubes_aem_v0.2_10.png)
+
+You will be prompted to set the SRK password, it is a password to access TPM's
+nonvolatile storage where the AEM secrets will be sealed.
+
+I have also shown the TPM properties to prove that we are using TPM 2.0. You
+can omit this. Then let's perform the AEM installation. In our example we are
+installing the AEM with MFA to the internal disk where Qubes OS is installed.
+This is generally not recommended, because using MFA and internal disk does
+not provide any extra security over not using MFA. But it is just for
+demonstration purposes. So in your case feel free to install AEM even on an
+external storage. In our case the boot partition is `/dev/sda1` (your may be
+different depending on where would you like to install AEM).
+
+```bash
+sudo anti-evil-maid-install -m /dev/sda1
+```
+
+![AEM installation 1](/img/qubes_aem_v0.2_11.png)
+![AEM installation 2](/img/qubes_aem_v0.2_12.png)
+
+If you chose MFA, then you will be presented a QR code to scan with an
+authentication app of your choice. it will show you a TOTP code later at boot
+process. This step will also ask you to provide a password for the secret and
+second time to provide any of the disk passwords for confirmation.
+
+The system now is ready to perform AEM boot, so reboot your machine!
+Now you should be presented with a SRK password prompt like this:
+
+![AEM SRK password prompt](/img/qubes_aem_v0.2_13.png)
+
+And later with the standard disk password prompt:
+
+![AEM disk password prompt](/img/qubes_aem_v0.2_14.png)
+
+After that AEM will print the messages saying, that the secrets have been
+sealed to the LUKS PCR 13 DRTM PCRs 17 and 18:
+
+![AEM secrets sealing](/img/qubes_aem_v0.2_15.png)
+
+You should see the above screen only once (unless you change something with
+GRUB, Xen or Dom0 kernel/initrd). Now that the AEM knows what the final PCR
+values should be, after next reboot AEM will successfully unseal the secrets
+it just sealed before a moment, and present a TOTP code. But before that, save
+the current PCR values by invoking the following command in Dom0 terminal:
+
+```bash
+sudo tpm2_pcrread sha256
+```
+
+![TPM PCRs](/img/qubes_aem_v0.2_16.png)
+
+Note the PCR values somewhere and reboot the machine. If everything went
+correctly, the PCRs should not change after reboot (you can comapre by reading
```

Comment:
```suggestion
correctly, the PCRs should not change after reboot (you can compare by reading
```

Reply:
This sentence has been removed: https://github.com/3mdeb/news-and-ideas/pull/543/commits/23be3297f4ca42f04cfc9231a9242525abf008d2#diff-c2e0e25a477d5cc5bf3075df551332eee15aaba457379ef35e8547639f285757L330